workers/unity/Packages/com.improbable.gdk.tools/GdkToolsConfiguration.cs

```diff
@@ -134,6 +158,30 @@ private static GdkToolsConfiguration CreateInstance()
             return config;
         }
 
+        void RetrieveAvailableiOSSimulators()
```

Comment:
I don't think this and `GetSimulatorUID` belongs here

Reply:
where would you put it?

Reply:
A static class somewhere with signatures like:

```
string GetSimulatorUID(int simulatorNameId);
string[] GetAvailableiOSSimulators();
```

This class should be purely configuration that should be committed.

Reply:
bump

Reply:
ah ignore the GdkToolsConfiguration.cs file for now,it will be removed after clean up :)